packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -53,10 +53,31 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
 
   public render(): JSX.Element {
     this._adjustProps();
-    const { data, hideNumberDisplay, hideLegend, theme, styles, barBackgroundColor, href, ignoreFixStyle, hideDenominator } = this.props;
+    const {
+      data,
+      benchmarkData,
+      targetData,
+      hideNumberDisplay,
+      hideLegend,
+      theme,
+      styles,
+      barBackgroundColor,
+      href,
+      ignoreFixStyle,
+      hideDenominator
+    } = this.props;
     const { palette } = theme!;
     const barHeight = ignoreFixStyle || data!.chartData!.length > 2 ? this.props.barHeight : 8;
-    const bars = this._createBarsAndLegends(data!, barHeight!, palette, barBackgroundColor, href);
+    if (benchmarkData) {
+      benchmarkData.color = benchmarkData.color || palette.neutralTertiary;
```

Comment:
Who's the designer for this? Is it possible for us to use bodySubtext and bodyText for this and target below? They'd go slightly darker.

Reply:
Additionally, please move style arguments into styleProps that are passed to the .styles.ts, instead of setting it in code. The .styles.ts should hold all the default colors that is uses if none are passed in.

Reply:
@joetheday is the designer for this. 
Semantically speaking, benchmark and target are graphical decorations. So I think it's okay to use the palette color. 

benchmarkColor is used to render color for both benchmark triangle and benchmark legend. So we can't directly move the default colors to the .styles.ts.
![image](https://user-images.githubusercontent.com/12230173/65299700-28130c80-dba3-11e9-82da-8b4d9b1e7b20.png)

Reply:
I'm going to have somebody look into this while I'm away. Stand by.

Reply:
bodyText and bodySubtext were what I was leaning towards as well, but held off since they're not text element.

I'm okay with base theme palette colors in this case, since yes, they're graphical decorations rather than something we have a semantic slot for.

Reply:
They play the same role as text elements. They are meant to be legible and denote meaning. We use text slots for icons and glyphs, for example, and this is basically a glyph.

Anyways, I'm okay with them being palette or bodyText slots, it's your call.